app/models/jurisdiction.rb

```diff
@@ -13,14 +13,6 @@ class Jurisdiction < ApplicationRecord
 
   has_many :analytics, class_name: 'Analytic'
 
-  scope :leaf_nodes, lambda {
```

Comment:
these were removed because of the following deprecation warning:

```
DEPRECATION WARNING: Class level methods will no longer inherit scoping from `leaf_nodes` in Rails 6.1. To continue using the scoped relation, pass it into the block directly. To instead access the full set of models, as Rails 6.1 will, use `Jurisdiction.default_scoped`. (called from block in <class:Jurisdiction> at /home/runner/work/SaraAlert/SaraAlert/app/models/jurisdiction.rb:17)
```